### PR TITLE
enable configuring trt options

### PR DESCRIPTION
**Pull request by askhade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,6 @@ TensorRT and OpenVino support:
 ```
 $ mkdir build
 $ cd build
-$ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_ONNXRUNTIME_VERSION=1.6.0 -DTRITON_BUILD_CONTAINER_VERSION=21.02 -DTRITON_ENABLE_ONNXRUNTIME_TENSORRT=ON -DTRITON_ENABLE_ONNXRUNTIME_OPENVINO=ON -DTRITON_BUILD_ONNXRUNTIME_OPENVINO_VERSION=2021.1 ..
+$ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_ONNXRUNTIME_VERSION=1.8.0 -DTRITON_BUILD_CONTAINER_VERSION=21.06 -DTRITON_ENABLE_ONNXRUNTIME_TENSORRT=ON -DTRITON_ENABLE_ONNXRUNTIME_OPENVINO=ON -DTRITON_BUILD_ONNXRUNTIME_OPENVINO_VERSION=2021.2.200 ..
```
Comment (askhade) [inline README.md:80]: -DTRITON_BUILD_ONNXRUNTIME_OPENVINO_VERSION=2021.2.200 (needs verification, working on this)

Comment (deadeyegoodwin) [reply README.md:80]: We build with 2021.2.200. You can see this in server/build.py

File: README.md [hunk 2]
```diff
@@ -44,14 +44,14 @@ Runtime version and a Triton container version that you want to use
...
-build.py](https://github.com/triton-inference-server/server/blob/r21.02/build.py#L60).
+in. For example, to build the ONNX Runtime backend for Triton 21.05,
+use the versions from TRITON_VERSION_MAP in the [r21.05 branch of
+build.py](https://github.com/triton-inference-server/server/blob/r21.05/build.py#L66).
 
 ```
 $ mkdir build
 $ cd build
-$ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_ONNXRUNTIME_VERSION=1.6.0 -DTRITON_BUILD_CONTAINER_VERSION=21.02 ..
+$ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_ONNXRUNTIME_VERSION=1.8.0 -DTRITON_BUILD_CONTAINER_VERSION=21.06 ..
```
Comment (deadeyegoodwin) [inline README.md:54]: 21.05, 21.06 not yet released

File: README.md (hunk 1 above)
Comment (deadeyegoodwin) [inline README.md:80]: 21.05

File: src/onnxruntime.cc [hunk 3]
```diff
@@ -239,9 +239,67 @@ ModelState::LoadModel(
...
+                    0,        // trt_dla_core
+                    0,        // trt_dump_subgraphs
+                    0,        // trt_engine_cache_enable
+                    nullptr,  // trt_engine_cache_path
+                    0,        // trt_engine_decryption_enable
+                    nullptr,  // trt_engine_decryption_lib_path
+                    0         // trt_force_sequential_engine_build
+                };
+                // Validate and set parameters
+                triton::common::TritonJson::Value params;
```
Comment (deadeyegoodwin) [inline src/onnxruntime.cc:264]: Add a "parameters" section to the README to describe these parameters. For example https://github.com/triton-inference-server/openvino_backend#using-the-openvino-backend

File: src/onnxruntime.cc [hunk 4]
```diff
@@ -263,8 +321,19 @@ ModelState::LoadModel(
       }
 
       // Default GPU execution provider.
-      RETURN_IF_ORT_ERROR(OrtSessionOptionsAppendExecutionProvider_CUDA(
-          soptions, instance_group_device_id));
+      // Using default values for evrything other than devide id and cuda stream
```
Comment (deadeyegoodwin) [inline src/onnxruntime.cc:324]: "device"

File: README.md (hunk 1 above)
Comment (askhade) [reply README.md:80]: right! I am testing whether we can increment ort to 1.8.0 without changing this version...

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

File: src/onnxruntime.cc (hunk 3 above)
Comment (askhade) [reply src/onnxruntime.cc:264]: thanks! I will add it... I was also looking for a place to document this. Currently I am adding it here: https://github.com/triton-inference-server/server/blob/main/docs/optimization.md#onnx-with-tensorrt-optimization

**Review by GuanLuo — DISMISSED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Review by deadeyegoodwin — DISMISSED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**